apps/datahub-e2e/src/e2e/datasets.cy.ts

```diff
@@ -445,15 +444,15 @@ describe('datasets', () => {
           .as('initialResultTitles')
         cy.get('@sortBy').selectDropdownOption('desc,createDate')
       })
-      it('changes the results order', () => {
-        cy.get('@initialResultTitles').then((initialResultTitles) => {
-          cy.get('@results')
-            .find('[data-cy="recordTitle"]')
-            .then(($titles) =>
-              $titles.toArray().map((title) => title.innerText.trim())
-            )
-            .should('not.eql', initialResultTitles)
-        })
+      it('changes the results order', function () {
+        cy.get('@results')
+          .find('[data-cy="recordTitle"]')
+          .then(($titles) => {
+            const titles = $titles
+              .toArray()
+              .map((title) => title.innerText.trim())
+            assert.notEqual(titles, this.initialResultTitles) // @initialResultTitles
```

Comment:
Why do we need to access `initialResultTitles` in the `this` scope here? I guess `.should('not.eql', @initialResultTitles)` is not the same?

Reply:
This is a subtlety of cypress: doing `cy.get('@myAlias')` will recompute the chain for the alias, whereas doing `this.myAlias` will return the value that was computed earlier. See:
https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Avoiding-the-use-of-this

Reply:
Ah, thanks for the explanations :-)